testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -196,7 +196,7 @@ def test_project_compare_projections(u_fresh):
     project1 = pca.project_single_frame(1)
 
     u_fresh.trajectory[0]
-    coord0 = project0(u_fresh.trajectory.ts).positions
+    coord0 = project0(u_fresh.trajectory.ts).positions.copy()
```

Comment:
Even if we do not change the ts.copy() in the DCDReader for now, I'd like to keep the copy() here for best practice demonstration and so that the tests could function with any reader and also for 3.0.

(Side note: it's confusing that `ts.positions` as a raw array behaves different from `ag.positions` as a view... I think the only (lame) excuse is that users aren't really supposed to play with `ts.positions` directly.)

Reply:
Roger that.